sql-cli/sql_cli/__main__.py

```diff
@@ -308,8 +308,8 @@ def init(
     help="""Deploy workflows to Astro Cloud.""",
 )
 def deploy(
-    workflow_name: str = typer.Option(  # skipcq: PYL-W0613
-        default=None,
+    workflow_name: str = typer.Argument(  # skipcq: PYL-W0613
+        default=...,
```

Comment:
@feluelle Why do we need to make it required? I was thinking that if no value is passed it means generate and deploy all workflows.

Reply:
Ah you are right 👍. But then we should already implement that here or?

Reply:
Let's do it separately. WDYT?

Reply:
No, if we keep it to None it accepts a parameter or None is fine too. If None then we generate all workflows and when a specific workflow is given it generates that specific. So my suggestion was we keep it as it is to None instead of making it required with `...`

Reply:
Sorry 🤦 I thought we would do the generate in this function.